core/assets/bundles/bundle_fr.properties

```
@@ -1890,24 +1890,24 @@ onset.mine.mobile = Tapez pour miner le \uf748 [accent]béryllium[] contenu dans
 onset.research = Ouvrez \ue875 l'arbre technologique.\nRecherchez et placez un \uf73e [accent]condenseur à turbine[] sur l'évent.\nCela va générer de [accent]l'énergie[].
 onset.bore = Recherchez et placez une \uf741 [accent]foreuse à plasma[].\nElle va automatiquement miner les ressources contenues dans les murs.
 onset.power = Pour [accent]alimenter[] la foreuse à plasma, recherchez et placez un \uf73d [accent]transmetteur à rayons[].\nConnectez le condenseur à la foreuse.
-onset.ducts = Recherchez et placez des \uf799 [accent]conduits[] pour déplacer les ressources minées depuis la foreuse vers le noyau.\nCliquez et faites glisser pour placer plusieurs conduits.\n[accent]Scrollez[] pour les faire pivoter.
-onset.ducts.mobile = Recherchez et placez des \uf799 [accent]conduits[] pour déplacer les ressources minées depuis la foreuse vers le noyau.\n\nRetenez votre doigt pendant une seconde et déplacez-le pour placer plusieurs conduits.
+onset.ducts = Recherchez et placez des \uf799 [accent]conduits[] pour déplacer les ressources minées depuis la foreuse vers le noyau.\n[accent]Cliquez et faites glisser[] pour placer plusieurs conduits.\n[accent]Scrollez[] pour les faire pivoter.
+onset.ducts.mobile = Recherchez et placez des \uf799 [accent]conduits[] pour déplacer les ressources minées depuis la foreuse vers le noyau.\n\n[accent]Retenez votre doigt[] pendant une seconde et [accent]déplacez-le[] pour placer plusieurs conduits.
 onset.moremine = Étendez vos exploitations minières.\nPlacez plus de foreuses à plasma et utilisez des transmetteurs à rayons pour les relier.\nMinez 200 minerais de béryllium.
 onset.graphite = Les blocs plus complexes requièrent du \uf835 [accent]graphite[].\nPlacez quelques foreuses à plasma pour miner du graphite.
 onset.research2 = Commencez à rechercher des [accent]usines[].\nRecherchez le \uf74d [accent]broyeur de parois[] et le \uf779 [accent]four de silicium[].
-onset.arcfurnace = le four de silicium a besoin de \uf834 [accent]sable[] et de \uf835 [accent]graphite[] pour créer du \uf82f [accent]silicium[].\nDe [accent]l'énergie[] est aussi requise.
+onset.arcfurnace = Le four de silicium a besoin de \uf834 [accent]sable[] et de \uf835 [accent]graphite[] pour créer du \uf82f [accent]silicium[].\nDe [accent]l'énergie[] est aussi requise.
 onset.crusher = Utilisez des \uf74d [accent]broyeurs de parois[] pour miner du sable.
-onset.fabricator = Utilisez des [accent]unités[] pour explorer la carte, défendre vos constructions et attaquer l'ennemi. Recherchez et placez un \uf6a2 [accent]fabricateur de tanks[].
+onset.fabricator = Utilisez des [accent]Unités[] pour explorer la carte, défendre vos constructions et attaquer l'ennemi. Recherchez et placez un \uf6a2 [accent]Tabricateur de Tanks[].
```

```suggestion
onset.fabricator = Utilisez des [accent]Unités[] pour explorer la carte, défendre vos constructions et attaquer l'ennemi. Recherchez et placez un \uf6a2 [accent]Fabricateur de Tanks[].
```
Attention faute de frappe.